### PR TITLE
doc: fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
Fixed some typos

### CODE REVIEW

**Comment by fanquake** [comment]
You cannot fix typos in subtrees, so please remove all changes to them. Feel free to send the changes upstream.

File: test/lint/lint-spelling.ignore-words.txt [hunk 1]
```diff
@@ -11,5 +11,4 @@ nin
 ser
 unparseable
 unser
-useable
```
Comment (MarcoFalke) [inline test/lint/lint-spelling.ignore-words.txt:14]: https://www.merriam-webster.com/dictionary/useable

Also, if you want to modify libraries, you will have to do it upstream

Comment (fanquake) [inline test/lint/lint-spelling.ignore-words.txt:14]: Not sure why you've dropped this and changed usages to `usable`, so you can revert that as well.

Comment (jbampton) [reply test/lint/lint-spelling.ignore-words.txt:14]: Most of the spelling in the codebase uses `usable` not `useable`.

Screen Shot 2021-05-07 at 9 51 10 pm

File: contrib/guix/guix-clean [hunk 2]
```diff
@@ -15,7 +15,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/libexec/prelude.bash"
 ###################
 
 ################
-# Required non-builtin commands should be invokable
+# Required non-builtin commands should be invocable
```
Comment (flack) [inline contrib/guix/guix-clean:18]: is that really a typo? invokable seems to exist: https://en.wiktionary.org/wiki/invokable (and that site lists invocable as an alternative spelling)

**Review by jarolrod — COMMENTED** [review]
Concept 0 % s/contruct/construct

`Invokable` and `useable` or both acceptable alternatives to `invocable` and `usable`. Their usage is correct and you're just swapping out one correct version for another. Bitcoin contributions come from all over the world, no need to impose the 'American' version of words.

**Comment by MarcoFalke** [comment]
Thank you for your contribution. However, this patch seems controversial, so I am going to close for now.

**Pull request closed without merging**